dll/win32/shell32/shlfolder.cpp

```diff
@@ -274,6 +274,42 @@ HRESULT SHELL32_CompareDetails(IShellFolder2* isf, LPARAM lParam, LPCITEMIDLIST
     return MAKE_COMPARE_HRESULT(ret);
 }
 
+HRESULT SHELL_CompareAllFields(IShellFolder* psf,
+                                 LPCITEMIDLIST pidl1, LPCITEMIDLIST pidl2,
+                                 UINT Count, int Skip)
+{
+    for (UINT i = 0; i < Count; ++i)
+    {
+        if (Skip == (int)i)
+            continue;
+        HRESULT hr = psf->CompareIDs(i, pidl1, pidl2);
+        if (hr && SUCCEEDED(hr))
+            return hr;
+    }
+    return MAKE_COMPARE_HRESULT(0);
+}
+
+HRESULT SHELL32_FolderImplCompareIDsTiebreaker(IShellFolder2* psf, LPARAM lParam,
```

Comment:
It would be nice to have doxygen comment about the purpose of this function. (So that you don't be the only guy here knowing what this "tiebreaking" thing is all about.)